### DISCUSSION TITLE
FWIW find out which code load `.env` into `os.environ`

### DISCUSSION THREAD

**Discussion by mkhludnev** [Cinnamon/kotaemon#242] [Show and tell]
I were troubleshoot some environment issues. 
import https://github.com/Cinnamon/kotaemon/blob/069f0f3c838622bbdc693ee49cd5f57bdabd400e/libs/kotaemon/kotaemon/storages/vectorstores/milvus.py#L4 causes a little bit invasive footprint on os.environ https://github.com/milvus-io/pymilvus/blob/87da0f100bf9078b4e5dcf8400eb79ed4956b88e/pymilvus/settings.py#L9
The possible drawback: 

- kotaemon calls graphrag as subprocess. I'm not sure, but loaded .env might be propagated to graphrag. Not sure it might be a problem
- a user might rely on this behavior, but it might be broken if .env is renamed

Just FYI/

**Comment by taprosoft** [comment]
Noted. This is an issue with pymilvus implementation and we will check if it affects GraphRAG env settings.
